horreum-web/src/domain/runs/DatasetComparison.tsx

```diff
@@ -213,28 +214,53 @@ type ViewComparisonProps = {
 
 function ViewComparison({headers, view, datasets, alerting}: ViewComparisonProps) {
     const [loading, setLoading] = useState(false)
-    const [rows, setRows] = useState<IRow[]>()
+    const [rows, setRows] = useState<IRow[]>([])
     const token = useSelector(tokenSelector)
+
     useEffect(() => {
         setLoading(true)
         Promise.all(datasets.map(ds => datasetApi.getSummary(ds.id, view.id)))
             .then(
                 summaries => {
-                    setRows(
-                        view.components.map(vc => ({
-                            cells: [
-                                vc.headerName,
-                                ...summaries.map(summary => {
-                                    const render = renderValue(
-                                        vc.render,
-                                        vc.labels.length == 1 ? vc.labels[0] : undefined,
-                                        token
-                                    )
-                                    return render(summary.view?.[vc.id], summary)
-                                }),
-                            ],
-                        }))
-                    )
+                    const rowsData: any[][] = view.components.map(vc => [
+                        vc.headerName,
+                        ...summaries.map(summary => {
+                            const render = renderValue(vc.render, vc.labels.length === 1 ? vc.labels[0] : undefined, token);
+                            return render(summary.view?.[vc.id], summary)
+                        })
+                    ])
+                    const allRows: IRow[] = []
+
+                    rowsData.forEach((rd, index) => {
+                        const isNum = rd.every((data, rdIndex ) => rdIndex === 0 || typeof rd[rdIndex] === 'number' )
+                        allRows.push({
+                            isOpen: isNum ? false: undefined,
+                            cells:  rd.map( (r, index) => ({
+                                    title: typeof r === "object" ? JSON.stringify(r) : r
```

Comment:
Thank you for sanitizing this but the more I think about it I think the value of `r` is the `v.headerName` or the output of the `render(...)` and I'm not sure if that is normally placed under the `title` for a `cell`. I'm going to read up on the `Table` `rows` attribute before we merge this

Reply:
https://www.patternfly.org/components/table/react-deprecated#table-rows
`IRowCell.title` is where the cell content is placed so this looks good to me.